### DISCUSSION TITLE
Add the ability to prevent jailed players from destroying blocks

### DISCUSSION THREAD

**Discussion by djnejk** [Zrips/CMI#4517] [Feature Requests]
Please add an option to prevent imprisoned players from destroying blocks

**Comment by CloudeLecaw** [comment]
Is this neessarry?
Most Servers create a little Subzone to rheir prison where the players cant build and usually this is enough ;)
your having no Protectionplugin?

1. Build a Pirson (of curse with blocks ;-) )
2. Create a zone, where players have no building rights
3. create a cmi prison with its cells 

Prison is not like cuff ;)
If u need what you want you also could set someone to prison and cuff him (and free and uncuff him) with a custom alias)

But that not how it should work.
Creat a zone(some call it also region) then take the prisoners there with cmis prison command. you can design as much cells as you want ... even if they are in the same room.

Without any protection plugin...well your prisoners break out of their cells ;-)

**Comment by djnejk** [comment]
I use a plugin on my survival server for a closed number of players and there are no territory protections, such as the residence plugin or the worldguard plugin. I believe that this option suits more people.

**Comment by CloudeLecaw** [comment]
hm but. most servers do that with a protection plugin ;)

well you may can build your prison at the world spawn. 
in serverproperties you can define a radius where only Operators could build. create your prison there if you dont wanne use a protection plugin. this area Counts for all worlds. servers that using protection plugins usually set it to 0 but in your case may a 10 or 20 would be nice so you have enough space arround the spawnplace of that world (not not the setspawn with cmi, the setworldspawn with the vanillacommand counts here)

Dont aspect that a nobuild thing will be added here...

**Comment by mrfloris** [comment]
Almost every server has worldguard installed, they could just make a region or warp for the jail. That way players can visit /warp jail still, but jailed players can't get out of their cells.

**Comment by CloudeLecaw** [comment]
I agrree with mrfloris. Nearly Esch Server hast a Protektion Plugin installed.worldguard, residence,redprotect,Grieg Prävention,and so on.

So i See this Features of adding Blockbreak to jails AS unnessary too
But the OP ist still freue to ASK for it, but i think this request here become forgotten ;)
